lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -18,6 +18,7 @@ class EngineParagraph implements ui.Paragraph {
     @required ui.TextAlign textAlign,
     @required ui.TextDirection textDirection,
     @required ui.Paint background,
+    @required List<ui.Shadow> shadows,
```

Comment:
This also needs to be added to `ui.ParagraphStyle`.

Reply:
done.